### DISCUSSION TITLE
[BUG] Multiple choice filter by job title

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#345] [4. Bug Reports] [answered]
Quería reportar un problema que tuve al usar  un multiple choice con filtro por job title. Tengo dos usuarios distintos con el job title de interviewer, pero cuando respondo el survey solo me aparece un solo usuario. Además intente agregar un nuevo usuario con el mismo jobtitle y sucede lo mismo.

Saludos!

**Comment by guillevalin** [comment, marked as answer]
@ValeMP esto es un problema conocido con el selector de usuarios, hay un trabajo pendiente en el roadmap para cambiar la forma en que se obtienen y muestran los usuarios en este tipo de pregunta del formulario el que esperamos sacar pronto en un futuro release.

**Reply by ValeMP** [reply]
Muchas gracias!
